jmbitcoin/jmbitcoin/secp256k1_main.py

```diff
@@ -396,8 +388,11 @@ def ecdsa_raw_sign(msg,
     else:
         #partial fix for secp256k1-transient not including customnonce;
         #partial because donations will crash on windows in the "if".
-        sig = newpriv.ecdsa_sign(msg, raw=rawmsg)
-    return newpriv.ecdsa_serialize(sig)
+        if rawmsg:
+            sig = newpriv.sign(msg, hasher=lambda x: x)
+        else:
+            sig = newpriv.sign(msg)
+    return sig
```

Comment:
Overall I agree that this code block is the correct way to replicate the behaviour.